model/src/main/kotlin/ScanResults.kt

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
Commit message: Maybe say "Also multiple scan results *for the same package*..."?

Reply:
Done.